modules/local/gzip_files.nf

```diff
@@ -23,7 +23,7 @@ process GZIP_FILES {
     gzip -c ${file_in} > ${output_name}
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        gzip: \$(echo \$(gzip --version 2>&1) | sed 's/^.*coreutils) //; s/ .*\$//')
+        gzip: \$(echo \$(gzip --help 2>&1 | head -n 1 ) | sed -e 's/ (.*//')
```

Comment:
This version of `gzip` in the container running this process does not have a `--version` flag. The closest is `--help`, which gives:

```
BusyBox v1.22.1 (2014-05-23 01:24:27 UTC) multi-call binary.

Usage: gzip [-cfd] [FILE]...

Compress FILEs (or stdin)

        -d      Decompress
        -c      Write to stdout
        -f      Force
```

This PR extracts the top line and uses that for the version.